code/modules/spells/spell_types/rod_form.dm

```diff
@@ -1,3 +1,6 @@
+/// The base distance a wizard rod will go without upgrades.
+#define BASE_WIZ_ROD_RANGE 13
```

Comment:
Can you put an `#undef BASE_WIZ_ROD_RANGE` at the end of the file, since it is not used elsewhere?

Reply:
Oops, forgot